aviary/interface/test/test_json.py

```diff
@@ -0,0 +1,58 @@
+import unittest
+from pathlib import Path
+
+from aviary.utils.functions import get_aviary_resource_path
+from openmdao.utils.testing_utils import require_pyoptsparse, use_tempdirs
+import aviary.api as av
+from aviary.interface.default_phase_info.height_energy import phase_info, phase_info_parameterization
+
+
+@use_tempdirs
+@require_pyoptsparse
+class TestJson(unittest.TestCase):
```

Comment:
From what I understand this is just checking that the json files _can_ be saved or loaded, and doesn't check that the files were properly created or that the off-design mission ran correctly - can we add comments explaining this, along with TODOs or an issue created to "finish" these tests and add a 2DOF test as well?

Reply:
this was mainly intended to catch the problem that EnumType wasn't added until 3.13, so these functions didn't work with the minimum versions we list, but since they weren't being tested anywhere, CI didn't catch that